src/Catel.MVVM/Catel.MVVM.Shared/Services/UIVisualizerService.wpf.cs

```diff
@@ -37,7 +37,7 @@ public partial class UIVisualizerService
         /// The default implementation returns the active window of the application.
         /// </summary>
         /// <returns>The active window.</returns>
-        protected virtual FrameworkElement GetActiveWindow()
+        protected virtual Window GetActiveWindow()
```

Comment:
I've been checking out the source code for MahApps (the Orchestra implementation), and MahApps doesn't let windows derive from the Window class. I think on a second thought it's better to keep this as FrameworkElement instead.